packages/gatsby/src/schema/resolvers.ts

```diff
@@ -16,6 +16,9 @@ import {
   SelectionNode,
   FieldNode,
 } from "graphql"
+import { Path } from "graphql/jsutils/Path"
+const reporter = require(`gatsby-cli/lib/reporter`)
```

Comment:
Do we want to use import too?
```suggestion
import reporter from  "gatsby-cli/lib/reporter`)
```